sdk/Directory.Build.props

```diff
@@ -2,12 +2,15 @@
   <PropertyGroup>
     <IsMgmtProject Condition="'$(IsMgmtProject)' == '' and $(MSBuildProjectName.Contains('.Management.'))">true</IsMgmtProject>
     <IsDataPlaneProject Condition="'$(IsDataPlaneProject)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Azure.')) and '$(IsMgmtProject)' != 'true'">true</IsDataPlaneProject>
-  </PropertyGroup>  
+  </PropertyGroup>
   <ItemGroup Condition="'$(IsTestProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
     <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
       <_Parameter1>SourcePath</_Parameter1>
       <_Parameter2>$(MSBuildProjectDirectory)</_Parameter2>
     </AssemblyAttribute>
   </ItemGroup>
   <Import Project="..\Directory.Build.props" />
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
+    <RunSettingsFilePath>$(RepoEngPath)\nunit.runsettings</RunSettingsFilePath>
```

Comment:
@heaths I assume this wouldn't impact the codeCoverage runsettings?

Reply:
Most likely, yes. RunSettings files don't merge.

Reply:
Actually, CC appears to be fine on the pipeline results, and the .runsettings are specific to VSTest, so maybe not. Let's try it and see?
```xml
    <VSTestSetting Condition="'$(VSTestSetting)' == ''">$(MSBuildThisFileDirectory)CodeCoverage.runsettings</VSTestSetting>
```

Reply:
Does it show for any of the pipelines being run in this PR? I thought it is only included for the release pipeline.

Reply:
No, it's all pipelines. The idea is to be able to see it on PRs so you can know if you're reducing CC. Right now, it's entirely manual to check (which I did).

Reply:
How do you check out of curiosity? for the release pipeline there is a tab but I don't see it for the others.

Reply:
Under checks, click on one of the test pipelines' Details link (any of them - they all go to the same results page for that pipeline), open the AzDO results page, click the back arrow to go the results summary, and you'll see:

![image](https://user-images.githubusercontent.com/1532486/112670753-8e26c280-8e1e-11eb-8b86-a437651ac3c4.png)